### ISSUE TITLE
Formation of multiple pronuclei during fertilization

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6811]
**Preferred term label:**
Formation of multiple pronuclei during fertilization

**Synonyms**

**Definition (free text, please give PubMed ID)**

Pronuclei formation is a critical process during fertilisation. Normally, there are two pronuclei (2PN), including the paternal pronucleus and the maternal pronucleus, in the zygote after fertilisation. It is generally accepted that multiple pronuclei (MPN) formation is due to the abnormal extrusion of the second polar body or to abnormal fertilisation with multiple sperm. The MPN therefore can cause infertility and recurrent failure of IVF/ICSI.

Three pro-nuclei (3PN) prevalence among all pregnancies has been estimated to be approximately 1% to 3%, whereas it accounts for 15%-18% of cytogenetically abnormal cases among spontaneous abortions.
PMID:31704776
PMID:26550358

**Parent term (use hpo.jax.org/app)**
Abnormal preimplantation embryonic development HP:0033335

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619176
PMID:31704776 2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
